types.go

```diff
@@ -5,7 +5,7 @@ import (
 	"io"
 	"time"
 
-	"github.com/docker/docker/pkg/units"
+	units "github.com/docker/go-units"
```

Comment:
you don't need to qualify the import, the package name is `units` already. It should be a drop in replacement.